spaces/S000136/properties/P000164.md

```diff
@@ -4,4 +4,4 @@ property: P000164
 value: true
 ---
 
-$|X| = 2^{2^\mathfrak{c}}$ is non-measurable
\ No newline at end of file
+$|X| = 2^{2^\mathfrak{c}}$ is smaller than the first measurable cardinal
```

Comment:
```suggestion
$|X| = 2^{2^\mathfrak{c}}$ is smaller than any measurable cardinal, if one exists.
```
Needs rephrasing because there may not be any measurable cardinal, let alone a first one.